python/pfs/pipe2d/gitChangelog.py

```diff
@@ -333,16 +375,102 @@ def process(self, repository_name):
 
         Returns
         -------
-        changelog : `dict` (`str`: `dict`(`str`: `set` (`str`)))
-            Maps a git tag to a ticket, and to a set of
-            repositories that have been updated for that
-            ticket.
+        changes : (`dict` (`str`: `str`), `dict`(`str`: `str`))
+            A tuple containing a mapping from tag to date,
+            and a mapping from ticket to date.
```

Comment:
Can you split this into two?

```
Returns
-------
tag_date : `dict` (`str`: `str`)
   Blah blah blah
ticket_date : `dict` (`str`: `str`)
   Yada yada yada
```

That makes clear that there are two things coming back.